pluma/schema/vr.py

```diff
@@ -91,63 +91,68 @@ def build_schema(root: Union[str, ComplexPath],
 
     # Glia streams
     streams.Glia.EyeTracking.Timestamps =         HarpStream(215, device='Glia', streamlabel='Glia.EyeTracking.Timestamps', root=root, autoload=autoload, parent_dataset=parent_dataset)
-    streams.Glia.EyeTracking.Data =               GliaStream('Glia/EyeTracking.bin',
-                                                             [glia_t_types,
-                                                              [('CombinedGaze.X', np.single), ('CombinedGaze.Y', np.single), ('CombinedGaze.Z', np.single), ('LeftOpenness', np.single), ('LeftOpennessConfidence', np.single), ('LeftDilation', np.single), ('LeftDilationConfidence', np.single), ('LeftPosition.X', np.single), ('LeftPosition.Y', np.single), ('RightOpenness', np.single), ('RightOpennessConfidence', np.single), ('RightDilation', np.single), ('RightDilationConfidence', np.single), ('RightPosition.X', np.single), ('RightPosition.Y', np.single)]],
-                                                             device='Glia',
-                                                             streamlabel='Glia.EyeTracking.Data',
-                                                             root=root, autoload=autoload,
-                                                             parent_dataset=parent_dataset)
+    streams.Glia.EyeTracking.Data =               ZeromqStream(['Glia/EyeTracking_Frame1.bin', 'Glia/EyeTracking_Frame2.bin'],
```

Comment:
I guess same for `PupilStream` we could perhaps have a `GliaStream` that already assumes some knowledge about the structure of glia data formats?